pyproject.toml

```diff
@@ -137,6 +137,11 @@ mypy_path = ["components", "bases"]
 files = ["bases/**/*.py", "components/**/*.py"]
 namespace_packages = true
 explicit_package_bases = true
+exclude = [
+  "components/renku_data_services/crc_migrations/versions",
```

Comment:
personally, I feel it's a bit suspicious if migrations aren't typed properly

Reply:
It was failing for missing annotations on stuff like `branch = None`. And we never used the variables where it was failing but I fixed it and removed the ignore statement in the pyproject file.